origin.spec

```diff
@@ -65,7 +65,11 @@ Obsoletes:      openshift < %{package_refector_version}
 ### AUTO-BUNDLED-GEN-ENTRY-POINT
 
 %description
-%{summary}
+Origin is a distribution of Kubernetes optimized for enterprise application
+development and deployment, used by OpenShift 3 and Atomic Enterprise. Origin
+adds developer and operational centric tools on top of Kubernetes to enable
+rapid application development, easy deployment and scaling, and long-term
+lifecycle maintenance for small and large teams and applications.
```

Comment:
Yes, first paragraph of https://github.com/openshift/origin/blob/master/README.md minus the first sentence.